var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -46,6 +46,11 @@ class Ncurses(AutotoolsPackage, GNUMirrorPackage):
         values=("none", "5", "6"),
         multi=False,
     )
+    variant(
+        "cxx",
+        default=True,
+        description="build c++ bindings",
+    )
```

Comment:
`ncurses` changes can be extracted in a separate PR

Reply:
Yup, I still need to extract this.